src/main/java/com/manardenza/dao/HotelDaoImpl.java

```diff
@@ -18,6 +19,14 @@ public static HotelDaoImpl getInstance() {
         return instance;
     }
 
+    public List<Hotel> getHotelsByName(String hotelName) {
+        return null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Return an empty collection instead of null. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1168)